Source/SDK/ClientBuilder.cs

```diff
@@ -238,6 +240,17 @@ public ClientBuilder WithLogging(ILoggerFactory factory)
             return this;
         }
 
+        /// <summary>
+        /// Sets the <see cref="JsonSerializerSettings"/> for serializing events.
```

Comment:
I suggest a corrected documentation 
```suggestion
        /// Sets a callback that configures the <see cref="JsonSerializerSettings"/> for serializing events.
```